demo/demo.js

```diff
@@ -182,7 +182,8 @@ var ContentKitDemo = exports.ContentKitDemo = {
       'simple-card': simpleCard,
       'edit-card': cardWithEditMode,
       'input-card': cardWithInput,
-      'selfie-card': selfieCard
+      'selfie-card': selfieCard,
+      'image': ContentKit.ImageCard
```

Comment:
Not in the editor constructor, but should be added to the default renderer (some version should be there)